.github/workflows/ci.yaml

```diff
@@ -504,12 +504,10 @@ jobs:
       # Verify that adding a plugin or rule produces clean code.
       - run: ./scripts/add_rule.py --name DoTheThing --prefix F --code 999 --linter pyflakes
       - run: cargo check
-      - run: cargo fmt --all --check
```

Comment:
I tried to make the `add_rule` output the right formatting but couldn't find the relevant lines and I then decided that I don't care enough. Just run `cargo fmt` and it will look correct.